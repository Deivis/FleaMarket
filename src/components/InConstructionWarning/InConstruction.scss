@import '../../styles/_variables';

.container__in-constuction {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-flow: row wrap;
  flex-grow: 1;
  color: $medium-grey;
}

.warning__in-constuction {
  font-size: 2.5em;
  padding: 0.5em;
  border-color: $medium-grey;
  border: 2px dashed;
}