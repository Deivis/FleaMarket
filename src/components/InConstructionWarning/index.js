import React from 'react';

import './InConstruction.scss';

const InConstruction = () => (
  <div className="container__in-constuction">
    <span className="warning__in-constuction"> In construction </span>
  </div>
);

export default InConstruction;
