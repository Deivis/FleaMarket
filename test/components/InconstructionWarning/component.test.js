import React from 'react';
import renderer from 'react-test-renderer';
import InConstructionWarning from '../../../src/components/InConstructionWarning';

describe('In construction Tests', () => {
  test('In construction Render', () => {
    const tree = renderer
      .create(<InConstructionWarning />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
