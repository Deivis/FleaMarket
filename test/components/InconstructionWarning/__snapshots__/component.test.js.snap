// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`In construction Tests In construction Render 1`] = `
<div
  className="container__in-constuction"
>
  <span
    className="warning__in-constuction"
  >
     In construction 
  </span>
</div>
`;
